### DISCUSSION TITLE
Vitepress deploy on vercel problem

### DISCUSSION THREAD

**Discussion by karakusnavy** [vercel/community#1201] [Help]
### Summary

when I deploy my Vitepress project on Vercel, it deploys incompletely. For example, a menu does not appear. Any ideas?

[when i deploy on vercel](https://i.stack.imgur.com/BIn3s.png)

[it's should be](https://i.stack.imgur.com/42PjL.png)

### Additional information

thanks.

i try redeploying, yarn..npm..

**Comment by amyegan** [comment]
🤔 Did you use the [VitePress template](https://vercel.com/new/clone?s=https://github.com/vercel/vercel/tree/main/examples/vitepress&template=vitepress&b=main&from=templates&teamCreateStatus=hidden) or the [VitePress "Getting Started" steps](https://vitepress.vuejs.org/guide/getting-started) to create the project? I want to be sure I'm looking at the right thing. If you have a public repo or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to share, that's even better. 🙏

**Comment by leerob** [comment]
Good news – we found a bug and have since fixed it! Could you try deploying VitePress again?

**Reply by karakusnavy** [reply]
> Good news – we found a bug and have since fixed it! Could you try deploying VitePress again?

Still not working..

**Reply by leerob** [reply]
Ahh, my apologies. I assumed this was a different issue. Reviewing your images more closely, this seems like an application-level issue. What happens when you do a production build locally?

**Reply by karakusnavy** [reply]
It works when I build locally. but build on vercel is not working. Do you want me to give you repo access?

**Reply by amyegan** [reply]
If you can share the repo or deployment link, I'm happy to help with debugging

**Reply by karakusnavy** [reply]
Too late.. I'am now using Nextra. Thanks.

**Comment by venkatmidhunmareedu** [comment]
I actually had a problem like that.... so i got a solution for you. I previously named the folder as app. And when i renamed it to docs and also edited the package.json file to 
```
"docs:dev": "vitepress dev docs",
"docs:build": "vitepress build docs",
"docs:preview": "vitepress preview docs"
```

from app -> docs 

then it worked for me.....
